### DISCUSSION TITLE
Balance Not Showing, but Acctualy is have Balance

### DISCUSSION THREAD

**Discussion by nzf210** [WalletConnect/walletconnect-monorepo#2115] [Developer Support]
Hii all, i have Problem when i use wallet connect using ScanQR, or i open with browser address like metamask browser and coinbase browser in phone balance not showing, this problem not all chain i get it for, polygon main, polygon mumbai, bsc testnet, for bsc main not problem and can't call payable function due to balance not detect, but i get balance use hook useBalance i get it, what wrong???

https://drive.google.com/file/d/1ZXBDJoCemGjwkbgt-gN9R8H10uTzD1Wk/view?usp=sharing

**Comment by ombayoeco11** [comment]
we're not Sherlock Holmes nor a Mighty All Know. Without providing the codes, how can we analyze whats wrong with it.
